be/src/storage/rowset/array_column_iterator.cpp

```diff
@@ -153,6 +153,11 @@ Status ArrayColumnIterator::next_batch(const vectorized::SparseRange& range, vec
 
         RETURN_IF_ERROR(_array_size_iterator->seek_to_ordinal_and_calc_element_ordinal(r.begin()));
         size_t element_ordinal = _array_size_iterator->element_ordinal();
+        // if array column in nullable or element of array is empty, element_read_range may be empty.
+        // so we should reseek the element_ordinal
```

Comment:
If we can add a test case to the integration case to avoid the problem.

Reply:
I will add some test cases into StarrocksTest after this pr is merged